js/bitget3.js

```diff
@@ -0,0 +1,2899 @@
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const type = this.safeString (params, 'type', market['type']);
+        if (type === undefined) {
+            throw new ArgumentsRequired (this.id + " fetchOrder() requires a type parameter (one of 'spot', 'swap').");
+        }
+        let method = undefined;
+        const request = {};
+        if (type === 'spot') {
+            const clientOid = this.safeString (params, 'client_oid');
+            if (clientOid !== undefined) {
+                method = 'apiPostOrderOrdersClientOid';
+                request['client_oid'] = clientOid;
+            } else {
+                method = 'apiPostOrderOrdersOrderId';
+                request['order_id'] = id;
+            }
+            request['method'] = 'getOrder';
+        } else if (type === 'swap') {
+            method = 'swapGetOrderDetail';
+            request['symbol'] = market['id'];
+            request['orderId'] = id;
+        }
+        const query = this.omit (params, 'type');
+        const response = await this[method] (this.extend (request, query));
+        //
+        // spot
+        //
+        //     {
+        //         "status":"ok",
+        //         "ts":1595897886716,
+        //         "data":{
+        //             "account_id":"7420922606",
+        //             "amount":"0.1000000000000000",
+        //             "canceled_at":"1595818631541",
+        //             "created_at":"1595792595897",
+        //             "filled_amount":"0.000000000000",
+        //             "filled_cash_amount":"0.000000000000",
+        //             "filled_fees":"0.000000000000",
+        //             "finished_at":"1595818631541",
+        //             "id":"671368296142774272",
+        //             "price":"150.000000000000",
+        //             "source":"接口",
+        //             "state":"canceled",
+        //             "symbol":"eth_usdt",
+        //             "type":"buy-limit"
+        //         }
+        //     }
+        //
+        //
+        // swap
+        //
+        //     {
+        //         "symbol":"cmt_ethusdt",
+        //         "size":"1",
+        //         "timestamp":"1595896459890",
+        //         "client_oid":"58775e54-0592-491c-97e8-e2369025f2d1",
+        //         "createTime":"1595885404607",
+        //         "filled_qty":"0",
+        //         "fee":"0",
+        //         "order_id":"671757564085534713",
+        //         "price":"150",
+        //         "price_avg":"0",
+        //         "status":"-1",
+        //         "type":"1",
+        //         "order_type":"0",
+        //         "totalProfits":"0"
+        //     }
+        //
+        const data = this.safeValue (response, 'data', response);
+        return this.parseOrder (data, market);
+    }
+
+    async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
```

Comment:
This is not working on my side: 
![image](https://user-images.githubusercontent.com/43336371/152379046-1c22772d-e6c9-4cb8-a862-f16e037ea9ee.png)
Can you point me to the docs of this endpoint, didn't find them

Reply:
Fixed in PR #11907